CHANGELOG.md

```diff
@@ -17,6 +17,8 @@ This section is for maintaining a changelog for all breaking changes for the cli
 ### Added
 - Document HTTP/2 support ([#330](https://github.com/opensearch-project/opensearch-java/pull/330))
 - Expose HTTP status code through `ResponseException#status` ([#756](https://github.com/opensearch-project/opensearch-java/pull/756))
+- Added missing WrapperQuery accessors and builder methods ([#806](https://github.com/opensearch-project/opensearch-java/pull/806))
```

Comment:
Actually I merged this too quickly. This changelog line should be added to the [Unreleased 2.x] section since this is to be released in 2.x. This way, the backport workflow also will not have merge conflicts.